tools/api-markdown-documenter/src/test/snapshots/html/simple-suite-test/sparse-config/test-suite-a/testfunctionreturninginlinetype-function.html

```diff
@@ -1,42 +1,21 @@
-<!DOCTYPE html>
+<!doctype html>
 <html lang="en">
   <head>
-    <meta charset="utf-8" />
+    <meta charset="utf-8">
   </head>
   <body>
     <section>
-      <h2>
-        testFunctionReturningInlineType
-      </h2>
+      <h2>testFunctionReturningInlineType</h2>
       <section>
-        <p>
-          Test function that returns an inline type
-        </p>
+        <p>Test function that returns an inline type</p>
       </section>
       <section>
-        <h3 id="testfunctionreturninginlinetype-signature">
-          Signature
-        </h3>
-        <code>
-          export declare function testFunctionReturningInlineType(): {
-          <br>
-              foo: number;
-          <br>
-              bar: TestEnum;
-          <br>
-          };
-        </code>
+        <h3 id="testfunctionreturninginlinetype-signature">Signature</h3><code>export declare function testFunctionReturningInlineType(): {<br>foo: number;<br>bar: TestEnum;<br>};</code>
```

Comment:
Oh, multi-line `<code>` requires `<br>` then? Still, if we can configure to force those to not be single-line I think it'd produce more readable HTML. Won't block on it if we can't, though.

Reply:
Not sure `hast-util-format` lets you customize to that degree, but I'll check.

Reply:
No, it doesn't. But I still prefer to go this route over managing our own rendering. If a user doesn't like this formatting, they can choose their own `hast` renderer to use.

Reply:
Test collateral has been updated. Confirmed that mutli-line code blocks preserve the necessary `<br>` tags.